src/components/DesktopPatientTable/index.js

```diff
@@ -43,7 +52,7 @@ const InfoWrapper = styled.div`
   display: flex;
   align-items: center;
   flex-wrap: wrap;
-  @media (max-width: 768px) {
+  position:relative @media (max-width: 768px) {
```

Comment:
```suggestion
  position:relative;
  @media (max-width: 768px) {
```

Reply:
fixed: the position has been removed as it is not relevant now